README.md

```diff
@@ -34,12 +34,27 @@ Learn more at [ipfs.io](https://ipfs.io) (it is really cool, we promise!)
 
 #### Automagical Detection of IPFS Resources
 
-  Requests for resources at IPFS-like paths (`/ipfs/$cid` or `/ipns/$peerid_or_fqdn`) are detected on any website.  
+  Requests for IPFS-like paths (`/ipfs/$cid` or `/ipns/$peerid_or_fqdn-with-dnslink`) are detected on any website.  
   If tested path is a valid IPFS address it gets redirected and loaded from a local gateway, e.g:  
   `https://ipfs.io/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`  
   → `http://127.0.0.1:8080/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`
 
-#### IPFS Status and Action Menu
+#### IPFS API exposed as `window.ipfs`
+
+A safe subset of IPFS APIs of user's node is exposed as `window.ipfs` on every webpage.  
+This means websites can detect if `window.ipfs` exists and opt-in to use it instead of spawning own `js-ipfs` node.  This saves resources, battery etc.
```

Comment:
"instead of spawning own" -> "instead of creating their own"

Reply:
"This saves resources, battery etc." -> "It saves system resources and battery (on mobile), avoids the overhead of peer discovery/connection, enables shared repository access and more!"